src/protocols/vnc/vnc.c

```diff
@@ -385,6 +385,17 @@ void* guac_vnc_client_thread(void* data) {
     }
 #endif
 
+    /* Disable remote console (Server input) */
+    if (settings->disable_server_input) {
+        rfbSetServerInputMsg msg;
+        msg.type = rfbSetServerInput;
+        msg.status = 1;
+        msg.pad = 0;
```

Comment:
FYI - I'm sure this is fine as-is (all structure members are explicitly initialized here), but you can do this quickly and cleanly in one step with:

```c
rfbSetServerInputMsg msg = {
    .type = rfbSetServerInput,
    .status = 1
};
```

with `pad` implicitly zeroed.

I'm OK with things as you have them, but just wanted to point the above out. We have additional syntax at our disposal here thanks to the codebase being written for C99.